### PR TITLE
fixes #1584 - bulk owner change

### PR DESCRIPTION
**Pull request by timogoebel**
This PR adds the functionality to bulk change the owner of hosts.

image

### CODE REVIEW

**Comment by timogoebel** [comment]
Katello tests are failing, I think this is unrelated and the usual random. [test]

File: test/functional/hosts_controller_test.rb [hunk 1]
```diff
@@ -368,6 +368,17 @@ def setup_multiple_environments
     assert Host.find(@host2.id).environment == environments(:global_puppetmaster)
   end
 
+  test "user with edit host rights with update environments should change owner" do
+    @request.env['HTTP_REFERER'] = hosts_path
+    assert_equal users(:admin).id_and_type, @host1.is_owned_by
```
Comment (domcleal) [inline test/functional/hosts_controller_test.rb]: `@host1` and 2 are populated by the `setup_multiple_environments` helper.  I'd suggest just putting the FG.create_list call (minus `:environment`) into your test.

File: test/functional/hosts_controller_test.rb [hunk 2]
```diff
@@ -368,6 +368,17 @@ def setup_multiple_environments
     assert Host.find(@host2.id).environment == environments(:global_puppetmaster)
   end
 
+  test "user with edit host rights with update environments should change owner" do
```
Comment (domcleal) [inline test/functional/hosts_controller_test.rb]: a) "with update owners"
b) it's in the middle of the two update environment tests, please move it below

File: test/functional/hosts_controller_test.rb (hunk 1 above)
Comment (timogoebel) [inline test/functional/hosts_controller_test.rb]: @domcleal: Yes, sorry for the mistake. Working on this and pushed to early. Just one more sec...

**Comment by timogoebel** [comment]
@domcleal : Fixed and moved the test.

**Comment by domcleal** [comment]
Merged as a92ee5b, thanks @timogoebel!

**Pull request closed without merging**